packages/ui/src/common/providers/modal/types.ts

```diff
@@ -35,7 +35,7 @@ export interface UseModal<Data> {
   modal: string | null
   modalData: Data
   showModal: <Call extends AnyModalCall>(call: Call) => void
-  hideModal: () => void
+  hideModal: (showConfirmOrAny?: boolean | any) => void
```

Comment:
Is this still necessary ? Is the solution in `@/common/providers/modal/provider.tsx` not working for the announcing candidacy and add new proposal cases ?